pkg/template/funcmap.go

```diff
@@ -50,6 +51,8 @@ func funcMap() template.FuncMap {
 		"fromJsonArray": fromJSONArray,
 		"ipNet":         ipNet,
 		"ipAt":          ipAt,
+
+		"semverCompare": semverCompare, // custom semver compare to avoid error
```

Comment:
don't override the default template function, cause this method may be called in other packages， such as `pkg/buildimage`

Reply:
But this will cause error, when `semver` is not valid. What I override is return (false,nil) to continue if passed in version is invalid statement, but not (false, err) to break all progress.
Any other suggesttions?